packages/holocron-module-route/__tests__/ModuleRoute.spec.jsx

```diff
@@ -13,24 +13,24 @@
  */
 
 import React from 'react';
-// eslint erroneously believes enzyme is not a dev dependency
-// eslint-disable-next-line import/no-extraneous-dependencies -- only used in tests
-import { shallow } from 'enzyme';
 
 import ModuleRoute from '../src/ModuleRoute';
+// eslint erroneously believes '@testing-library/react' is not a dev dependency
+// eslint-disable-next-line import/no-extraneous-dependencies -- only used in tests
+const { render } = require('@testing-library/react');
```

Comment:
is this lint error only in your editor or does it show up when running eslint in a terminal? Sometimes I find the editor gets confused but actual linter doesn't

Reply:
this is an eslint error